core/src/main/java/io/questdb/MessageBus.java

```diff
@@ -33,10 +34,7 @@
 
 public interface MessageBus extends Closeable {
     @Override
-    default void close() {
-        // We need to close only queues with native backing memory.
-        Misc.free(getTableWriterEventQueue());
-    }
+    void close();
```

Comment:
This method declaration is redundant since the interface extends `Closeable`.

Reply:
I usually leave these methods as a way of removing `IOException`, which otherwise has to be handles where interface is used. It can be removed in this particular instance by virtue of fact `MessageBus` interface is not closed via interface. There are other instances of this "trick" on other interfaces

Reply:
Yes, concrete implementation is free to remove the exception on its own. I got the intention , so let's keep things consistent.